src/azure-cli-core/azure/cli/core/context.py

```diff
@@ -8,36 +8,59 @@
 
 from azure.cli.core._config import (CONTEXT_CONFIG_DIR, GLOBAL_CONFIG_DIR)
 
+
 class ContextNotFoundException(Exception):
+
     def __init__(self, context_name):
         super(ContextNotFoundException, self).__init__(context_name)
         self.context_name = context_name
+
     def __str__(self):
         return "The context '{}' does not exist.".format(self.context_name)
 
+
 class ContextExistsException(Exception):
+
     def __init__(self, context_name):
         super(ContextExistsException, self).__init__(context_name)
         self.context_name = context_name
+
     def __str__(self):
         return "The context '{}' already exists.".format(self.context_name)
 
+
 class CannotDeleteActiveContextException(Exception):
+
     def __str__(self):
         return "You cannot delete the current active context"
 
+
 class CannotDeleteDefaultContextException(Exception):
+
     def __str__(self):
         return "You cannot delete the default context."
 
+
 ACTIVE_CONTEXT_FILE = os.path.join(GLOBAL_CONFIG_DIR, 'active_context')
 ACTIVE_CONTEXT_ENV_VAR_NAME = 'AZURE_CONTEXT'
 DEFAULT_CONTEXT_NAME = 'default'
 
-get_active_context = lambda: get_context(get_active_context_name())
-context_exists = lambda n: any([x for x in get_context_names() if x == n])
-get_context_file_path = lambda n: os.path.join(CONTEXT_CONFIG_DIR, n)
-get_contexts = lambda: [get_context(context_name) for context_name in get_context_names()]
+
+def get_active_context():
+    return get_context(get_active_context_name())
+
+
+def context_exists(n):
+    return any([x for x in get_context_names() if x == n])
+
+
+def get_context_file_path(n):
+    return os.path.join(CONTEXT_CONFIG_DIR, n)
+
+
+def get_contexts():
+    return [get_context(context_name) for context_name in get_context_names()]
+
```

Comment:
I don't see how the method-based versions are superior to the lambda versions. Is there a performance difference?